aiofreepybox/api/player.py

```diff
@@ -0,0 +1,227 @@
+from typing import Any, Dict, List, Optional
+
+from aiofreepybox.access import Access
+
+_DEFAULT_PLAYER_API_VERSION = "v6"
+
+
+class Player:
+    """
+    Player
+    """
+
+    def __init__(
+        self, access: Access, player_api_version: Optional[str] = None
+    ) -> None:
+        self._access = access
+        self._player_api_version = (
+            _DEFAULT_PLAYER_API_VERSION
+            if player_api_version is None
+            else player_api_version
+        )
+
+    media_control_seek_args = {"seek_position": 0, "type": "seek_position"}
+    media_control_stream = {"quality": "", "source": ""}
+    media_control_stream_args = {"stream": media_control_stream, "type": "stream"}
+    media_control_track_args = {"track_id": 0, "type": "track_id"}
+    media_control_command = {
+        "play",
+        "pause",
+        "play_pause",
+        "stop",
+        "next",
+        "prev",
+        "seek_forward",
+        "seek_backward",
+        "seek_to",
+        "repeat_all",
+        "repeat_one",
+        "repeat_off",
+        "repeat_toggle",
+        "shuffle_on",
+        "shuffle_off",
+        "shuffle_toggle",
+        "record",
+        "record_stop",
+        "select_audio_track",
+        "select_srt_track",
+        "select_stream",
+    }
+    media_control_data_schema = {"args": media_control_stream_args, "cmd": "pause"}
+
+    async def _get_d_p_id(self) -> Optional[int]:
+        """
+        Get default player id
+        """
+
+        player = await self.get_players()
+        if player is not None:
+            return player[0]["id"]
```

Comment:
```suggestion
        players = await self.get_players()
        if players is not None:
            return players[0]["id"]
```
to make it clearer that this is a list?

Reply:
Maybe remove `if players is not None:` because it should be normal to get an error if we have no players, or throw a `FreeboxNoPlayerException` instead of returning `None`.

So we don't need the 8 following `if player_id is not None:`